app/api/healthcheck/route.ts

```diff
@@ -1,43 +1,37 @@
 import { NextRequest, NextResponse } from 'next/server';
 import { execSync } from 'child_process';
+import { string } from 'zod';
 
 // GET method to retrieve health status and git information
 export async function GET(request: NextRequest) {
   try {
     const currentDateTime = new Date().toISOString();
     
-    // Get git commit SHA and time for main branch
-    let gitCommitSha = 'unknown';
-    let gitCommitTime = 'unknown';
-    
-    try {
-      // Get the latest commit SHA on main branch
-      gitCommitSha = execSync('git rev-parse HEAD', { 
-        encoding: 'utf8',
-        cwd: process.cwd()
-      }).trim();
-      
-      // Get the commit time for the latest commit
-      gitCommitTime = execSync('git log -1 --format=%cd --date=iso', { 
-        encoding: 'utf8',
-        cwd: process.cwd()
-      }).trim();
-    } catch (gitError) {
-      console.warn('Could not retrieve git information:', gitError);
-      // Keep default values if git commands fail
+    // Get git commit SHA for deployment
+    let gitCommitSha = process.env.VERCEL_GIT_COMMIT_SHA;
+    let healthData;
+
+    if (gitCommitSha && typeof gitCommitSha === 'string' && gitCommitSha.length > 2) {
```

Comment:
@minump If the goal is to do a minimal sanity check of the Sha, I recommend updating to the length check from `> 2` to `=== 40`.